skxray/core/accumulators/histogram.pyx

```diff
@@ -186,8 +185,8 @@ class Histogram:
         # numerical data type
         numtypes = [np.dtype(int), np.dtype(float)]
         numtypeindex = {tp : i for i, tp in enumerate(numtypes)}
-        ctypeindices = [numtypeindex.get(c.dtype, MAX_DIMENSIONS)
-                for c in coords]
+        ctypeindices = [numtypeindex.get(c.dtype, len(numtypes)+1)
```

Comment:
I think this should be something greater than len(numtypes)-1. I put len(numtypes) but forgot we have zero based indexing here from enumerate. Do you agree?